rclrs/src/wait.rs

```diff
@@ -275,6 +313,7 @@ impl WaitSet {
                     .push(Arc::clone(&subscription.waitable));
             }
         }
+
```

Comment:
There should be a block similar to subscriptions/clients/services for guard conditions as well.

Reply:
I thought about it some more, and I agree we can add it back if we can make guard conditions implement `Eq`, so that you can find out _which_ guard condition triggered. That might make it useful.